CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 10. Added an HTML report [#44](https://github.com/Plant-Food-Research-Open/genepal/issues/44)
 11. Added content type as text/html for the MultiQC and genepal reports
 12. Added sra-tools for RNASeq data download [#102](https://github.com/Plant-Food-Research-Open/genepal/issues/102)
+13. Added cDNA and cds outputs to <OUTPUT_DIR>/annotations/<SAMPLE> directory [#118](https://github.com/Plant-Food-Research-Open/genepal/issues/118)
```

Comment:
Can you kindly start change log for a new version: 0.6.0?

0.5.0 has already been released.